Sources/TokamakPublish/Plot+HTML.swift

```diff
@@ -17,18 +17,18 @@ import TokamakStaticHTML
 
 public extension Plot.Node {
   static func view<V>(_ view: V) -> Self where V: View {
-    .raw(StaticHTMLRenderer(view).html)
+    .raw(StaticHTMLRenderer(view).render())
   }
 
   static func view<V>(@ViewBuilder _ body: () -> V) -> Self
     where V: View
   {
-    .raw(StaticHTMLRenderer(body()).html)
+    .raw(StaticHTMLRenderer(body()).render())
   }
 }
 
 extension Plot.Node: View, AnyHTML where Context == Plot.HTML.BodyContext {
-  public var innerHTML: String? {
+  public func innerHTML(shouldSortAttributes: Bool) -> String? {
```

Comment:
Should this argument be present if it's unused? And if not, can it be changed back to a property again?

Reply:
I believe it has to have that argument to satisfy a requirement of [`AnyHTML`](https://github.com/TokamakUI/Tokamak/blob/main/Sources/TokamakStaticHTML/Views/HTML.swift)

Reply:
Plot (the package being extended here) posseses no such feature as attribute sorting

Reply:
Ah, I see, `render()` called here is coming from `Plot`, not `StaticHTMLRenderer`?

Reply:
That's correct

Reply:
If so, would you be able to add a comment by that `renderer()` call clarifying that `Plot` doesn't support attribute sorting? Such could would clarify why `shouldSortAttributes` argument is ignored, for posterity.

Reply:
Done!